src/chrome_binary.py

```diff
@@ -82,7 +82,13 @@ def download(url, name, base="."):
         revision = str(revision)
         binary_dir_path = os.path.join(path, revision)
         if os.path.exists(binary_dir_path):
-            return True
+            brp = self.get_browser_path(path, revision)
```

Comment:
Can you help me understand this change?

If the path exists, I thought we could assume the browser and driver exist?

Reply:
You're right. 
This change is unnecessary.
I changed this because before we use os.rename, there was a driver download error so I just want to verify and redownload the driver and browser.